plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -16,54 +18,76 @@ Even still, there are times when creating a custom table can be more effective.
 
 Now that you can identify when custom database tables are useful, let's learn how to add one!
 
-## The `wpdb` Class in PHP
+In this example, you're going to create a custom table to store form submissions, which will have a field for the name and a message. 
+
+## The `WPDB` class
+
+To start, you need to first establish a connection to the WordPress database. Thankfully, WordPress already does this automatically by using the `WPDB` class.
 
-To start, we need to first establish a connection to the WordPress database. Thankfully, WordPress already does this automatically by using the `wpdb` class.
+The `WPDB` class is used to manage a database connection while also providing helpful methods to perform queries and retrieve data from the connected database in a structured way. 
 
-The `wpdb` class is used to manage a database connection while also providing helpful methods to perform queries and retrieve data from the connected database in a structured way. In PHP, WordPress automatically sets the `$wpdb` global variable to an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
+During PHP execution, WordPress creates a `$wpdb` global object variable as an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
 
-You can access it in PHP like this:
+Because the `$wpdb` object exists in the global namespace, you can access it in PHP like this:
 
 ```php
 global $wpdb;
 ```
 
-The `$wpdb` instance also contains crucial information about the WordPress database which we'll need to properly create and manage our own custom tables. Let's see how as we create our first one.
+The `$wpdb` instance also contains crucial information about the WordPress database which you'll need to properly create and manage our own custom tables. Let's see how as we create our first one.
```

Comment:
Since we're saying "you", maybe the `Let's see how as we create our first one.` part doesn't make as much sense. I was struggling with a transitional phrase here, regardless. 😅

Reply:
:smile: I struggle with it too. In this case, I think it's fine. It's like you're guiding the learner along with you.

Reply:
Maybe something like this? 😅

> Let's see this in action as we create our first one.

It's the "see how as" part that sounds like incorrect grammar that's bugging me! Haha! It sounds natural when spoken/heard aloud but looks weird in writing/reading.

Also, since "we'll need" was changed to "you'll need", shouldn't we say "manage _**y**our_ own custom tables"?